api4/user.go

```diff
@@ -2381,3 +2384,101 @@ func verifyUserEmailWithoutToken(c *Context, w http.ResponseWriter, r *http.Requ
 
 	w.Write([]byte(user.ToJson()))
 }
+
```

Comment:
If we are planning to have more authentication providers on top of those two, what would be your opinion on having only one endpoint `migratie_auth` for all of the possible migrations instead of one per `to`?

Reply:
They don't share a common data type for requests. So, I believe it will result to slightly more complex code. But I could be missing something :)